sources/ClangSharp.PInvokeGenerator/PInvokeGeneratorConfigurationOptions.cs

```diff
@@ -58,5 +58,7 @@ public enum PInvokeGeneratorConfigurationOptions
         GeneratePreviewCode = 0x00800000,
 
         GenerateTemplateBindings = 0x01000000,
+
+        GenerateSourceLocationAttribute = 1 << 25,
```

Comment:
nit: This should be either `0x02000000` or all entries should be updated to use the `1 << n` syntax.

I don't have a particular preference for which is done, I'd just like for it to be consistent

Reply:
I'll update the rest to use bit shifting to be explicit and simplify visual validation of flag values.

Reply:
Done.